docs/helpers/env_vars.yaml

```
@@ -1,3 +1,104 @@
+ACTIVITYLOG_SERVICE_ACCOUNT_ID:
+  name: OCIS_SERVICE_ACCOUNT_ID;ACTIVITYLOG_SERVICE_ACCOUNT_ID
+  defaultValue: ""
+  type: string
+  description: The ID of the service account the service should use. See the 'auth-service'
+    service description for more details.
+  introductionVersion: "5.0"
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+ACTIVITYLOG_STORE:
+  name: OCIS_PERSISTENT_STORE;ACTIVITYLOG_STORE
+  defaultValue: nats-js-kv
+  type: string
+  description: 'The type of the store. Supported values are: ''memory'', ''ocmem'',
+    ''etcd'', ''redis'', ''redis-sentinel'', ''nats-js'', ''noop''. See the text description
+    for details.'
+  introductionVersion: pre5.0
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+ACTIVITYLOG_STORE_AUTH_PASSWORD:
+  name: OCIS_PERSISTENT_STORE_AUTH_PASSWORD;ACTIVITYLOG_STORE_AUTH_PASSWORD
+  defaultValue: ""
+  type: string
+  description: The password to authenticate with the store. Only applies when store
+    type 'nats-js-kv' is configured.
+  introductionVersion: "5.0"
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+ACTIVITYLOG_STORE_AUTH_USERNAME:
+  name: OCIS_PERSISTENT_STORE_AUTH_USERNAME;ACTIVITYLOG_STORE_AUTH_USERNAME
+  defaultValue: ""
+  type: string
+  description: The username to authenticate with the store. Only applies when store
+    type 'nats-js-kv' is configured.
+  introductionVersion: "5.0"
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+ACTIVITYLOG_STORE_DATABASE:
+  name: ACTIVITYLOG_STORE_DATABASE
+  defaultValue: activitylog
+  type: string
+  description: The database name the configured store should use.
+  introductionVersion: pre5.0
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+ACTIVITYLOG_STORE_NODES:
+  name: OCIS_PERSISTENT_STORE_NODES;ACTIVITYLOG_STORE_NODES
+  defaultValue: '[127.0.0.1:9233]'
+  type: '[]string'
+  description: A list of nodes to access the configured store. This has no effect
+    when 'memory' or 'ocmem' stores are configured. Note that the behaviour how nodes
+    are used is dependent on the library of the configured store. See the Environment
+    Variable Types description for more details.
+  introductionVersion: pre5.0
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+ACTIVITYLOG_STORE_SIZE:
+  name: OCIS_PERSISTENT_STORE_SIZE;ACTIVITYLOG_STORE_SIZE
+  defaultValue: "0"
+  type: int
+  description: The maximum quantity of items in the store. Only applies when store
+    type 'ocmem' is configured. Defaults to 512 which is derived from the ocmem package
+    though not exclicitely set as default.
```

```suggestion
    though not explicitly set as default.
```

I know that this text is being imported from elsewhere - so I suppose that I need to make a PR to the source, and then another PR can be done to fix the little stuff.